packages/fscomponents/src/components/ModalHalfScreen.tsx

```diff
@@ -93,16 +94,34 @@ export class ModalHalfScreen extends PureComponent<ModalHalfScreenProps, ModalHa
   }
 
   render(): JSX.Element {
+    let calculatedPosition;
+    const screenHeight = Dimensions.get('window').height;
```

Comment:
This should be implemented as a Dimensions listener so we don't have to recalculate this value with every render: https://github.com/brandingbrand/flagship/blob/3bfde0951d4a460f26ae2e283ae0996bae8c0e0a/packages/pirateship/src/screens/Category.tsx#L69

Reply:
Good stuff, thanks for catching that!  Implemented in updated PR.